doc/api/next_api_changes/2019-06-10-AL.rst

```diff
@@ -0,0 +1,7 @@
+RendererSVG now uses a single counter for externally written images
+```````````````````````````````````````````````````````````````````
+
+When using `RendererSVG` with ``rcParams["svg.image_inline"] ==
+True``, externally written images now use a single counter even if the
+``renderer.basename`` attribute is overwritten, rather than a counter per
+basename.
```

Comment:
Maybe add
~~~
This will not affect you unless you explicitly manipulate ``renderer.basename``.
~~~

We don't want to scare users unnecessarily. :smile: (I also would have been fine without a message. I was just asking if we are aware of that - not an expert on renderer stuff and its usage.)

Reply:
done. (indeed, I think this is the kind of API change notes that's more confusing than helpful, but heh.)